core/sequencer/sequencer.go

```diff
@@ -334,23 +347,82 @@ func (s *Sequencer) CreateEpoch(ctx context.Context, forceNewEpoch bool) error {
 	})
 	mapSetEnd := time.Now()
 	if err != nil {
-		return err
+		return nil, err
 	}
 	revision = setResp.GetMapRoot().GetMapRevision()
 	glog.V(2).Infof("CreateEpoch: SetLeaves:{Revision: %v, HighestFullyCompletedSeq: %v}", revision, seq)
 
 	// Put SignedMapHead in an append only log.
 	if err := queueLogLeaf(ctx, s.tlog, s.logID, setResp.GetMapRoot()); err != nil {
 		// TODO(gdbelvin): If the log doesn't do this, we need to generate an emergency alert.
-		return err
+		return nil, err
 	}
 
-	mutationsCtr.Add(float64(len(mutations)))
+	mutationsCtr.Add(float64(len(mRange)))
 	indexCtr.Add(float64(len(indexes)))
 	mapUpdateHist.Observe(mapSetEnd.Sub(mapSetStart).Seconds())
 	createEpochHist.Observe(time.Since(start).Seconds())
 	glog.Infof("CreatedEpoch: rev: %v, root: %x", revision, setResp.GetMapRoot().GetRootHash())
-	return nil
+
+	// MapRevisions start at 1. Log leave's index starts at 0.
+	// MapRevision should be at least 1 since the Signer is
+	// supposed to create at least one revision on startup.
+	respEpoch := setResp.GetMapRoot().GetMapRevision() - 1
```

Comment:
Add TODO referencing google/trillian#828.  This will cause the map revisions to also start at 0